omod/src/main/java/org/openmrs/module/attachments/fragment/controller/DashboardWidgetFragmentController.java

```diff
@@ -19,6 +20,12 @@
 	
 	public void controller(FragmentModel model, @FragmentParam("patient") PatientDomainWrapper patientWrapper, UiUtils ui,
 	        @InjectBeans AttachmentsContext context) {
+		// Ensure name is HTML-safe to prevent XSS in info messages
+		PersonName htmlSafePersonName = new PersonName(ui.escapeHtml(patientWrapper.getPatient().getGivenName()),
+		        ui.escapeHtml(patientWrapper.getPatient().getMiddleName()),
+		        ui.escapeHtml(patientWrapper.getPatient().getFamilyName()));
+		patientWrapper.getPatient().removeName(patientWrapper.getPersonName());
+		patientWrapper.getPatient().addName(htmlSafePersonName);
```

Comment:
Instead of calling `removeName` and then `addName` why not just add the `htmlSafePersonName` to the `jsonConfig` object directly on line 30?

Reply:
I was not sure if I could do that since it uses a set for the names and I wasn't sure how it would know which names were the HTML safe names when accessed. @isears

Reply:
Ok it looks like we would have to create a patient wrapper, which is probably unnecessary complexity here. Let's go with your original solution